index.js

```diff
@@ -8,7 +8,10 @@ var CacheStore = require('./cache_store');
 var DEFAULT_TOTAL_VNODES = 1024;
 var DEFAULT_CALM_THRESHOLD = 500;
 var DEFAULT_RETRY_INTERVAL_MS = 5000;
-var MAX_PARALLEL_TASKS = 10;
+
+// We're setting a modest limit to make sure we're not starving the event-loop of CPU
```

Comment:
Do you mean `saturating` instead of `starving` here?

Reply:
Starving feels more natural to me, since we're effectively taking this CPU from the event-loop